tests/lang_callable/macros/tfor_ill_formed.nim

```diff
@@ -0,0 +1,25 @@
+discard """
+  cmd: "nim check $options $file"
+  action: reject
+  nimout: '''
+tfor_ill_formed.nim(17, 14) Error: expression has no type: case true
+of true:
+  discard
+else:
+  discard
+'''
+"""
+import std/macros
+
+macro t1(): untyped =
```

Comment:
Can this not be a simple `for` loop rather than resorting to macros? Any `nkError` in the iterable position should do.

Reply:
am not sure, here just ensure it goes to there, not supressed by other error